ebml_matroska.xml

```diff
@@ -63,7 +63,7 @@
       </element>
       TimecodeScale When combined with <a href="http://www.matroska.org/technical/specs/index.html#TimecodeScaleDenominator">TimecodeScaleDenominator</a> the Timestamp scale is given by the fraction TimecodeScale/TimecodeScaleDenominator in seconds.-->
       <element name="Duration" level="2" id="0x4489" type="float" minver="1" range="&gt; 0x0p+0">
-        <documentation lang="en">Duration of the Segment (based on TimecodeScale).</documentation>
+        <documentation lang="en" type="definition">Duration of the Segment measured in nanoseconds when scaled by TimecodeScale.</documentation>
```

Comment:
Yes, but that's not in the scope of this pull request. I'd rather have one that only deals with timestamp vs. timecode usage.

Problem is if you rename elements building applications will break if the libraries follow suit. And if they don't there's discrepancy between the specs and the libraries that can be confusing.